projects.md

```diff
@@ -19,9 +19,11 @@ We have been compiling a set of [project guidelines](project_guidelines.html) th
 | --------| ------------- |----------|-----------|
 | [DD4hep](https://github.com/AIDASoft/DD4hep)   | Generic Detector Description toolkit for HEP  | LGPL v3 | [Markus Frank](mailto:marks.frank@cern.ch) |
 | [fads](https://pkg.go.dev/go-hep.org/x/hep/fads?tab=doc)          | FAst Detector Simulation  | BSD-3 | [Sebastien Binet](mailto:binet@cern.ch) |  
-| [Gaudi](http://gaudi.web.cern.ch/gaudi/)          | Event data processing framework | Unk | [Marco Clemencic](mailto:marco.clemencic@cern.ch) |  
+| [Gaudi](http://gaudi.web.cern.ch/gaudi/)          | Event data processing framework | Apache-2 | [Marco Clemencic](mailto:marco.clemencic@cern.ch) |  
 | [HepMC3](https://gitlab.cern.ch/hepmc/HepMC3) | C++ Event Record for Monte Carlo Generators | LGPL v3 | [Witek Pokorski](mailto:witold.pokorski@cern.ch) |
+| [Phoenix](https://github.com/HSF/phoenix) | Event display framework | Apache-2 | [Ed Moyse](mailto:Edward.Moyse@cern.ch) |
 | [podio](https://github.com/AIDASoft/podio) | Event Data Model Library based on plain-old-data | GPL v3 | [Benedikt Hegner](mailto:benedikt.hegner@cern.ch) |
 | [PM4hep](https://github.com/hegner/PM4hep) | Minimal Plugin Manager | GPL v3 | [Benedikt Hegner](mailto:benedikt.hegner@cern.ch) |
+| [prmon](https://github.com/HSF/prmon) | Standalong process resource consumption monitor | Apache-2 |  [Graeme Stewart](mailto:graeme.andrew.stewart@cern.ch) |
```

Comment:
Perhaps "standalone" @graeme-a-stewart ? 😄

Reply:
*kick's self...*

Yes, of course. Apologies to the gods of the commit history, but I wanted to get this in quickly for Phoenix. Too quickly it seems!